sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/PipelinedDocumentQueryExecutionContext.java

```diff
@@ -105,6 +107,26 @@ private static BiFunction<String, PipelinedDocumentQueryParams<Document>, Flux<I
         return createAggregateComponentFunction;
     }
 
+    private static BiFunction<String, PipelinedDocumentQueryParams<Document>, Flux<IDocumentQueryExecutionComponent<Document>>> createBaseHybridComponentFunction(
+        DiagnosticsClientContext diagnosticsClientContext,
+        IDocumentQueryClient client,
+        PipelinedDocumentQueryParams<Document> initParams,
+        DocumentCollection collection) {
+        CosmosQueryRequestOptions requestOptions = initParams.getCosmosQueryRequestOptions();
+        BiFunction<String, PipelinedDocumentQueryParams<Document>, Flux<IDocumentQueryExecutionComponent<Document>>> createBaseComponentFunction;
+
+        createBaseComponentFunction = (continuationToken, documentQueryParams) -> {
+            CosmosQueryRequestOptions orderByCosmosQueryRequestOptions =
+                qryOptAccessor.clone(requestOptions);
+
+            qryOptAccessor.getImpl(orderByCosmosQueryRequestOptions).setCustomItemSerializer(null);
```

Comment:
why forcing serializer to null? Can you please add one test case for hybrid search with a custom serializer?

Reply:
Removed the code to set customSerializer to null, will add the test case in a follow up PR.